controllers/bounce_processes.go

```diff
@@ -326,3 +336,76 @@ func getAddressesForUpgrade(logger logr.Logger, r *FoundationDBClusterReconciler
 
 	return addresses, nil
 }
+
+// checkIfClusterControllerNeedsRestart checks if the cluster controller must be restarted. There are a few cases where
+// this might be required. One case is when at least on tester process is running in the cluster and that tester process
+// fails. Currently this leads to the cluster controller reporting unreachable processes and the status incomplete message.
+// Having those messages in the cluster's machine-readable status could block some operations of the operator and the
+// solution to that is to restart the cluster controller process.
+func checkIfClusterControllerNeedsRestart(status *fdbv1beta2.FoundationDBStatus) *fdbv1beta2.ProcessAddress {
+	// If the status contains no cluster messages we can skip further check.
+	if len(status.Cluster.Messages) == 0 {
```

Comment:
We should add a special minimum uptime here for this behaviour to limit the CC restarts. This should be configurable over a command line flag on the operator with the default value of 1h.